podcasts/New Creation/New Preview/Rules/SmartPlaylistRulesView.swift

```diff
@@ -0,0 +1,188 @@
+import SwiftUI
+import PocketCastsDataModel
+
+struct SmartPlaylistRulesView: View {
+    @State var isExpanded: Bool = false
+    @EnvironmentObject var theme: Theme
+
+    let viewModel: PlaylistPreviewViewModel
+
+    var body: some View {
+        switch viewModel.playlistMode {
+        case .creation:
+            if viewModel.isInPreview {
```

Comment:
Could we split the two sides of the if in two separate View variables? To make it easier to understand the structure of the main view.